readme.md

```diff
@@ -110,6 +110,7 @@ User can choose between 2 types of installations:
 |MongoDB user's password|  **Autogenerate**  | ✓ | ✓ |
 |Redis password<sup>4</sup>|  **Autogenerate**  | ✓ | ✓ |
 |Use AWS storage|  **No**  | ✓ | ✓ (frontend only) |
+|Use WAL-E PostgreSQL backups<sup>5</sup> |  **No**  | ✓ | ✓ (frontend only) |
```

Comment:
I think it should be `backend only`. 
BTW, would you mind to remove (frontend only) for AWS storage. We do ask for AWS credentials in the backend if backups are activated (maybe another exponent <sup>n</sup> could explain it)